### ISSUE TITLE
Data type setting (inference_input_type, inference_output_type)

### ISSUE THREAD

**Issue by freesokim** [google-ai-edge/ai-edge-torch#670]
### Description of the bug:

As shown in the quantization method on the TensorFlow website,
(https://www.tensorflow.org/lite/performance/post_training_quantization)

converter.inference_input_type = tf.int8      # or tf.uint8 
converter.inference_output_type = tf.int8    # or tf.uint8

is there a way to set the inference_input_type and inference_output_type? 
If not, are there any plans to support this in the future?

**Comment by gaikwadrahul8** [comment]
Hi, @freesokim 
Thank you for showing your interest to support `inference_input_type` and `inference_output_type` like in TensorFlow's tf.lite.TFLiteConverter but as far I know the **ai-edge-torch** library facilitates the conversion of PyTorch models to TensorFlow Lite (TFLite) format for on-device deployment but there is no direct equivalent support to `converter.inference_input_type` and `converter.inference_output_type` as of now please refer this [official documentation ](https://github.com/google-ai-edge/ai-edge-torch/blob/main/docs/pytorch_converter/README.md#quantization) and [PyTorch forum discussion](https://discuss.pytorch.org/t/how-to-specify-input-and-output-types/180541) 

Thank you for your cooperation and understanding.

**Comment by freesokim** [comment]
Thank you for your response.

The method (tfl_converter_flags) you mentioned is already known to me. 
The model quantized using this method was different from the model quantized using PT2EQuantizer. 
In conclusion, the model quantized using PT2EQuantizer had higher accuracy, which is necessary for my situation. 
Therefore, in the current situation, I need a way to set the inference_input_type and inference_output_type to fp32 using the PT2EQuantizer method.

For reference, my quantization settings are as follows:
        pt2e_quantizer = PT2EQuantizer().set_global(
            get_symmetric_quantization_config(is_per_channel=True, is_dynamic=False))

Thanks.

**Comment by aravindmurali22** [comment]
Hi @freesokim  ,

Thank you for taking the time to report this and for your patience. Please let us know if you are experiencing this issue in the latest release of [litert-torch](https://github.com/google-ai-edge/litert-torch/releases/tag/v0.9.1). If the problem persists, please file a new issue with the updated details. This will help us track and investigate it using the most current context.
We will close this issue to keep our tracker aligned with the current codebase and release cycle.
Thank you again for your support and understanding!
